pom.xml

```diff
@@ -369,6 +369,7 @@
                 <module>persistence-modules/hibernate-ogm</module> <!-- hibernate-ogm wasn't updated because it doesn't support jakarta API -->
                 <module>persistence-modules/java-cassandra</module> <!-- JAVA-21464 cassandra-unit library doesn't support to run with jdk9 and above -->
                 <module>persistence-modules/spring-data-cassandra-reactive</module> <!--JAVA-21844-->
+				<module>persistence-modules/spring-data-neo4j</module>
```

Comment:
let's fix the formatting here to use 4 spaces instead of tabs

Reply:
I will fix the conflict and this will solve the space indentation :)